index-de-DE.html

```diff
@@ -0,0 +1,87 @@
+---
+title: Mu-An Chiou
+permalink: /de-DE/
+layout: default
+root: true
+---
+
+<p>
+  Eine Software-Designerin und -Entwickerin, die bescheidene Websites erstellt.
+</p>
+<p>
+  Jetzt in Berlin, Deutschland<span class="time" data-timestamp-text=", wo es gerade {time} ist"></span>. 
+  Frühere Wohnorte sind New York, London und Birmingham, UK.
+</p>
+
+<nav aria-label="Extra Inhalt">
+  <a href="{% link _pages/qna.md %}">Fragen und Antworten</a><hr class="inline-hr"> 
+  <a href="{% link _pages/blogroll.md %}">Blogroll</a><hr class="inline-hr">
+  <a href="{% link _pages/issues.html %}">Issues</a><hr class="inline-hr">
+  <a href="{% link _pages/banners.md %}">88&times;31</a>
+</nav>
+<nav aria-label="Soziale Links">
+  <a href="https://github.com/muan" rel="me">GitHub</a><hr class="inline-hr"> <a href="https://read.cv/muan" rel="me">Read.cv</a><hr class="inline-hr"> <a href="https://mastodon.social/@muan" rel="me">Mastodon</a><hr class="inline-hr"> <a href="https://www.duolingo.com/profile/muan" rel="me">Duolingo</a>
+</nav>
+
+<div class="lang">
+  <a href="/zh-TW/" srclang="zh-TW">正體中文</a><a href="/" srclang="en-US">English</a><a href="/de-DE/" srclang="de-DE" aria-current="true">Deutsch</a>
+</div>
+
+<hr class="hr">
+
+<h2 class="text">Stories</h2> <a href="/stories.xml" class="styled-link">RSS</a><a href="https://github.com/dddddddddzzzz/OpenStories" class="styled-link">Was is das?</a>
```

Comment:
```suggestion
<h2 class="text">Stories</h2> <a href="/stories.xml" class="styled-link">RSS</a><a href="https://github.com/dddddddddzzzz/OpenStories" class="styled-link">Was ist das?</a>
```

unless you want to be very colloquial?